lib/iris/fileformats/netcdf/loader.py

```diff
@@ -673,6 +691,8 @@ def __init__(self, var_dim_chunksizes=None):
 
         """
         self.var_dim_chunksizes = var_dim_chunksizes or {}
+        self.Modes = Enum("Modes", ["DEFAULT", "FROM_FILE", "AS_DASK"])
```

Comment:
This object never changes as a result of what's going on in the instance, so it should be a class variable instead - i.e. defined at the top of the class, not in `__init__()` - I think that's common practice for `Enum`s like this.